hot-reload-gradle-plugin/src/main/kotlin/org/jetbrains/compose/reload/arguments.kt

```diff
@@ -183,6 +184,7 @@ private class ComposeHotReloadArgumentsBuilderImpl(
             /* Non API elements */
             virtualMethodResolveEnabled = project.composeReloadVirtualMethodResolveEnabled,
             dirtyResolveDepthLimit = project.composeReloadDirtyResolveDepthLimit,
+            composeResourcesEnabled = project.composeReloadComposeReloadResourcesEnabled,
```

Comment:
composeReloadComposeReloadResourcesEnabled looks strange (repeated). Can it be changed?